application/src/objects/car.js

```diff
@@ -95,4 +97,32 @@ Car.prototype.applyForces = function()
     );
 };
 
+Car.prototype.fall = function()
+{
+    this.falling = true;
+
+    // Adjust car position slightly to avoid some cases where the falling
+    // effect is shown while not over a drop tile.
+    if (Math.abs(this.body.velocity.x) > Math.abs(this.body.velocity.y)) {
+        this.body.x += (50 * (this.body.velocity.x > 0 ? 1 : -1));
+    }
+    else if (Math.abs(this.body.velocity.y) > 0) {
+        this.body.y += (50 * (this.body.velocity.y > 0 ? 1 : -1));
+    }
```

Comment:
Since the car is still doing some weird stuff that sometimes makes it disappear into the sand, would you consider getting rid of this block for now until we can come back and address it more comprehensively?